src/index.html

```diff
@@ -6,9 +6,35 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <h1 class="visually_hidden">Moyo</h1>
+    <header class="header container">
+      <a href="#home">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="header_nav">
+        <ul class="header_nav__list">
+          <li class="header_nav__item" ><a class="header_nav_link is-active" href="#">Apple</a></li>
```

Comment:
Format your code as follow to improve its readability:
```suggestion
          <li class="header_nav__item">
            <a class="header_nav_link is-active" href="#">
              Apple
            </a>
          </li>
```
Fix all cases
![image](https://github.com/mate-academy/layout_moyo-header/assets/96409563/ea33e0e6-e3c7-4f97-9ade-383ab5b39181)